lib/units/validateBody.js

```diff
@@ -148,6 +148,27 @@ function getBodyValidator(realType, expectedType) {
   return [null, validator[0]];
 }
 
+/**
+ * Parses a given JSON Schema to ensure it's a valid JSON.
+ * @param {string} schema
+ */
+const validateJsonSchema = (schema) => {
+  try {
+    JSON.parse(schema);
+  } catch (error) {
+    throw new Error(`\
+Failed to parse a given JSON Schema:
+
+${schema}
+
+Please check if that is a valid JSON and a valid JSON Schema.
```

Comment:
I would rephrase this error, as there is no JSON Schema validation in place.

It should say something like
```
Please check that the given JSON Scheme is a valid JSON.
```

Reply:
Resolved in 7342a7a.